app/namespace_test.go

```diff
@@ -1063,6 +1064,117 @@ func (s *NamespaceTestSuite) TestClearCertificateFilters() {
 	}
 }
 
+func (s *NamespaceTestSuite) TestUpdateCodecServer() {
+	ns := "ns1"
+	type morphGetResp func(*namespaceservice.GetNamespaceResponse)
+	type morphUpdateReq func(*namespaceservice.UpdateNamespaceRequest)
+
+	tests := []struct {
+		args         []string
+		expectGet    morphGetResp
+		expectErr    bool
+		expectUpdate morphUpdateReq
+	}{
+		{
+
+			args:      []string{"namespace", "update-codec-server"},
+			expectErr: true,
+		},
+		{
+			args:      []string{"namespace", "update-codec-server", "--namespace", ns},
+			expectErr: true,
+		},
+		{
+			args:      []string{"n", "ucs", "-n", ns, "-endpoint", "fakehost:9999"},
+			expectGet: func(g *namespaceservice.GetNamespaceResponse) {},
+			expectUpdate: func(r *namespaceservice.UpdateNamespaceRequest) {
+				r.Spec.CodecSpec = &namespace.CodecServerPropertySpec{Endpoint: "fakehost:9999"}
+			},
+		},
+		{
+			args:      []string{"n", "ucs", "-n", ns, "-e", "fakehost:9999", "--pass-access-token"},
+			expectGet: func(g *namespaceservice.GetNamespaceResponse) {},
+			expectUpdate: func(r *namespaceservice.UpdateNamespaceRequest) {
+				r.Spec.CodecSpec = &namespace.CodecServerPropertySpec{
+					Endpoint:        "fakehost:9999",
+					PassAccessToken: true,
+				}
+			},
+		},
+		{
+			args:      []string{"n", "ucs", "-n", ns, "-e", "fakehost:9999", "--pat", "--include-credentials"},
+			expectGet: func(g *namespaceservice.GetNamespaceResponse) {},
+			expectUpdate: func(r *namespaceservice.UpdateNamespaceRequest) {
+				r.Spec.CodecSpec = &namespace.CodecServerPropertySpec{
+					Endpoint:           "fakehost:9999",
+					PassAccessToken:    true,
+					IncludeCredentials: true,
+				}
+			},
+		},
+		// {
```

Comment:
If this is not used any more, just remove all these commented code.

Reply:
Oops did not catch that, thank you!